apps/studio/components/interfaces/QueryPerformance/EnableIndexAdvisorButton.tsx

```diff
@@ -70,7 +72,8 @@ export const EnableIndexAdvisorButton = () => {
     }
   }
 
-  if (isAdvisorAvailable) return null
+  // if index_advisor is already enabled or not available to install, show nothing
+  if (isIndexAdvisorEnabled || !isIndexAdvisorAvailable) return null
```

Comment:
hmm technically don't need to check for `isIndexAdvisorAvailable`

since `isIndexAdvisorEnabled` already innately includes the check for `isIndexAdvisorAvailable`